nan.h

```diff
@@ -281,20 +281,49 @@ NAN_INLINE uint32_t NanUInt32OptionValue(
 template<typename T>
 v8::Local<T> NanNew(v8::Handle<T>);
 
-template<typename T>
-NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Handle<T> val) {
-  return NanNew(val);
-}
+namespace NanIntern {
```

Comment:
Good suggestion. The `NanIntern` namespace exists since before, but I'll rename them too.